openverse_catalog/dags/maintenance/pr_review_reminders/pr_review_reminders.py

```diff
@@ -139,6 +134,29 @@ def base_repo_name(pr: dict):
     return pr["base"]["repo"]["name"]
 
 
+branch_protection_cache = defaultdict(dict)
```

Comment:
Since this is a module-level variable, would you be alright making this all-caps so it's consistent with other module-level constants?

Reply:
Interesting, even though it's mutable? :thinking: It should have a leading underscore anyway, would that still be all caps then?

Reply:
Yes, especially because it's mutable! Referencing it anywhere will reference the single instantiation of it and mutate it. And yes please on all caps + underscore! Examples from `licenses.constants`: https://github.com/WordPress/openverse-catalog/blob/277de7a861b863178869dd0bbb93b06f503f074a/openverse_catalog/dags/common/licenses/constants.py#L3

Reply:
Done in https://github.com/WordPress/openverse-catalog/pull/642/commits/29d36664849d18b08cb36d19f5a36975dbf3d91a